rero_ils/modules/documents/templates/rero_ils/detailed_view_documents.html

```diff
@@ -193,5 +193,18 @@ <h5 class="d-inline-block">{{ _('Export Formats') }}:</h5>
 {{ super() }}
 {% assets "rero_ils_documents_detailed_js" %}
 <script src="{{ ASSET_URL }}"></script>
+
+<script type="text/javascript">
+$( window ).load(function() {
+    /* Store state of view tab */
+    $('a[data-toggle="tab"]').on('click', function(e) {
+      localStorage.setItem('activeTabDoc', $(this).attr('href'));
+    });
+    var activeTab = localStorage.getItem('activeTabDoc');
+    if(activeTab) {
+      $('.nav-tabs a[href="' + activeTab + '"]').tab('show');
+    }
+});
+</script>
```

Comment:
Shouldn't this JS be part of an asset, like the one at line 194 ?

Reply:
If I put it in the asset it is not working any more. Someone with more knowledge has to find the correct way to implement this.

Reply:
Ok, then let's ask somebody: @AoNoOokami @zannkukai @sebastiendeleze @jma ! :slightly_smiling_face:

Reply:
This will not work in production as security does not allows to have a JS code in a page.